connector-splunk/src/main/java/com/redhat/cloud/notifications/connector/splunk/SplunkExceptionProcessor.java

```diff
@@ -0,0 +1,41 @@
+package com.redhat.cloud.notifications.connector.splunk;
+
+import com.redhat.cloud.notifications.connector.ExceptionProcessor;
+import io.quarkus.logging.Log;
+import org.apache.camel.Exchange;
+import org.apache.camel.http.base.HttpOperationFailedException;
+
+import static com.redhat.cloud.notifications.connector.splunk.ExchangeProperty.ACCOUNT_ID;
+
+public class SplunkExceptionProcessor extends ExceptionProcessor {
+
+    private static final String DEFAULT_LOG_MSG = "Message sending failed on %s: [orgId=%s, accountId=%s, historyId=%s, targetUrl=%s]";
+    private static final String HTTP_LOG_MSG = "Message sending failed on %s: [orgId=%s, accountId=%s, historyId=%s, targetUrl=%s] " +
+            "with status code [%d] and body [%s]";
+
+    @Override
+    protected void process(Throwable t, Exchange exchange) {
```

Comment:
This method replaces [eventing-integrations/ErrorHandlingRoutes.java#L47-L99](https://github.com/RedHatInsights/eventing-integrations/blob/3faa2fb8c58002ce5473166ade7181ece4fb14ed/splunk-quarkus/src/main/java/com/redhat/console/integrations/ErrorHandlingRoutes.java#L47-L99)